### DISCUSSION TITLE
accounting for sex as a covariate in getMarkerFeatures()

### DISCUSSION THREAD

**Discussion by alyamahmoud** [GreenleafLab/ArchR#1205] [Questions / Documentation] [answered]
Hi there, 

Is there a way I can include 'sex' as a blocking factor when doing pairwise testing with getMarkerFeatures? The bias term would take only numeric values. 

Thank you

**Comment by rcorces** [comment, marked as answer]
I've never done this but presumably you could just make sex a numeric?

**Reply by alyamahmoud** [reply]
Thank you, that seemed to work. However, I was not getting any significant hits when I blocked sex + wilcoxon but I get quite some (~15% up and down) when block sex + binomial. This is for differential peak testing. Does not it make more sense to use binomial testing for testing for peaks?

**Reply by alyamahmoud** [reply]
I realize this might be a different question/thread.

**Reply by rcorces** [reply]
I'm not sure I would say it makes more sense to use any specific test. They each have rationales for use. Wilcoxon is certainly more stringent. Like any analysis, you should make sure you believe that there is a real difference in what is being called significant.

That being said, we are working on revamping some of the differential testing based on the findings of [this publication](https://www.nature.com/articles/s41467-021-25960-2.pdf)

**Reply by alyamahmoud** [reply]
Thank you! this is helpful.

**Comment by DaianeH** [comment]
Hi there,
On the same note, I would like to make "Patient", or sample name, a covariable since some of our AML patients have high number of CNVs that makes them have particular and unique patterns of open chromatin, and I would like to control for those. It's straightforward to dummy code binary variables like Sex, but what do you advise in the case of categorical variables such as Patient, when you have let's say 15 patients?
Thank you in advance,
